### ISSUE TITLE
Menubar is invisible in vscode 

### ISSUE THREAD

**Issue by hrmon** [ubuntu/yaru#1060]
Hi!
I installed yaru on Ubuntu 18.04 via snap. 
The problem I am facing is that in vscode the menu bar is not visible, although it is working.
In my system, I only see this problem in vscode and other apps are okay. Also when the window is not in focus the menu bar is visible.

![screenshot from 2018-12-24 17-10-19](https://user-images.githubusercontent.com/10273568/50400974-89d35100-07a0-11e9-9cad-31d7119c7c76.png)

communitheme info:
```
name:      communitheme
summary:   The next Ubuntu theme built by the community.
publisher: Didier Roche (didrocks)
license:   unset
description: |
  Yaru, formerly known as Communitheme, is the new Ubuntu theme built by the
  community. Yaru will become the default Ubuntu theme in Ubuntu 18.10. This
  package allows you to try out the theme on Ubuntu 18.04 LTS.
  
  To try out the theme, install this package on Ubuntu 18.04 LTS, restart
  your computer and select the "Ubuntu with communitheme snap" session from
  the login screen.
  
  More information is available at
  https://community.ubuntu.com/t/faq-ubuntu-new-theme/1930.
snap-id:      Yd6CISPIf6tEf3ZEJ0cqSoEg9rG2VkRi
tracking:     stable
refresh-date: yesterday at 05:20 +0330
channels:                    
  stable:    0.1 (1364) 18MB -
  candidate: ↑               
  beta:      ↑               
  edge:      0.1 (1469) 19MB -
installed:   0.1 (1364) 18MB -
```

vscode info:
```
Version: 1.29.1
Commit: bc24f98b5f70467bc689abf41cc5550ca637088e
Date: 2018-11-15T19:07:43.495Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64
```

**Comment by Feichtmeier** [comment]
Yes that's known and I think there is also an issue for that on vscode

We have a black menu bar with white menu on the light theme
And electron can't handle this combination currently

**Comment by hrmon** [comment]
By now the custom menubar in vscode has no problem

**Comment by clobrano** [comment]
According to [this](https://github.com/ubuntu/yaru/issues/969) closed ticket a first fix in Electron was released in version 3.0.1 (which does not fix completely, but still), while the version @hrmon is using is older (2.0.12) so it is still shows the old problem.

Closing this then
